recipes/ninja/all/conanfile.py

```diff
@@ -51,7 +51,7 @@ def package(self):
 
     def package_info(self):
         # ensure ninja is executable
-        if str(self.settings.os_build) in ["Linux", "Macosx"]:
+        if str(self.settings.os_build) in ["Linux", "Macos"]:
```

Comment:
Normally, conan catches this kind of errors.
It did not here, because the setting was converted to a string.
```suggestion
        if self.settings.os_build in ["Linux", "Macos"]:
```

Reply:
Thanks I'm kind of new to conan so I didn't know there was error checking like that 👍